firmware/controllers/sensors/sensor.h

```diff
@@ -117,7 +117,8 @@ class Sensor {
 	SensorType m_type;
 
 	// Get this sensor's index in the list
-	constexpr size_t getIndex() {
+	// constexpr size_t getIndex() {
```

Comment:
@DonaldBecker @mck1117 please help me understand this change?

Reply:
I'm not sure why `constexpr` got removed either

Reply:
The compiler on my machine (recently downloaded from launchpad.net), claimed that it wasn't confirmed a constexpr.

Reply:
I have `gcc version 7.4.0 (Ubuntu 7.4.0-1ubuntu1~18.04.1)`
It's possible that a newer compiler is stricter about it.

Reply:
@mck1117 would you be interested to try gcc 8?

Reply:
what do the official builds run?

Reply:
@mck1117 my build server runs gcc 8.3.1

Reply:
@mck1117 @donaldbecker that's outdated since https://launchpad.net/gcc-arm-embedded/+announcement/15345 :)

Reply:
So clearly it works on 8.3, because the build server doesn't choke on it.  I can try 9.x at some point and see what happens.

Reply:
Trying it with:
`gcc version 9.2.1 20191025 (release) [ARM/arm-9-branch revision 277599] (GNU Tools for Arm Embedded Processors 9-2019-q4-major)`
Compiles but doesn't link.